react/src/attention/AttentionPattern.tsx

```diff
@@ -164,15 +165,37 @@ export function AttentionPattern({
 
   return (
     <Col>
-      <Row style={{ aspectRatio: showAxisLabels ? undefined : "1/1" }}>
-        <Chart
-          type="matrix"
-          options={options}
-          data={data}
-          width={1000}
-          height={1000}
-          updateMode="none"
-        />
+      <Row>
+        <div
+          style={{
+            // Chart.js charts resizing is weird.
+            // Responsive chart elements (which all are by default) require the
+            // parent element to have position: 'relative' and no sibling elements.
+            // There were previously issues that only occured at particular diplay
+            // sizes and zoom levels. See:
+            // https://github.com/alan-cooney/CircuitsVis/pull/63
+            // https://www.chartjs.org/docs/latest/configuration/responsive.html#important-note
+            // https://stackoverflow.com/a/48770978/7086623
+            position: "relative",
+            // Set the maximum width of zoomed heads such that a head with just a
+            // few tokens doesn't have crazy large boxes per token and the chart
+            // doesn't overflow the screen. Other heads fill their width.
+            maxWidth: zoomed
```

Comment:
What's the reason for only setting a max width for zoomed heads? 

I'm imagining using this method on a two-head model with a two-token input, it seems like in that case the unzoomed heads could end up being too wide.

Reply:
I'm mostly just copying what was there before. But the zoomed head has a max width so the boxes don't get weirdly big. The non-zoomed heads don't need this because they are in a smaller container (which they should fill).

It also seems best not to set a max width for the non-zoomed heads, so this function can be more easily used in a new function without hard-to-understand behavior.

Example for 1 and 2 head models:
![Screenshot 2023-07-08 at 23 26 55](https://github.com/alan-cooney/CircuitsVis/assets/47218308/c4e5dd93-b20b-4e6b-b363-b4487a2e7167)

![Screenshot 2023-07-08 at 23 29 51](https://github.com/alan-cooney/CircuitsVis/assets/47218308/e9f4dd62-81e5-45a1-a854-568d61b52772)